slsReceiverSoftware/include/HDF5FileStatic.h

```diff
@@ -675,7 +672,7 @@ class HDF5FileStatic: public virtual slsDetectorDefs {
 
 				//source file name
 				std::string srcFileName = HDF5FileStatic::CreateFileName(fpath, fnameprefix, findex,
-						frindexenable, framesSaved, dindex, numunits, i);
+						framesSaved, dindex, numunits, i);
```

Comment:
framesSaved shouldn't be used to create the file name anymore. It should be the hyperslab loop variable "j" most likely. It looks like you test hdf5, but I don't understand how it found the source files to create the virtual one. Did it create the virtual files then?

Reply:
Good catch! changed it to j